pages/blog.html

```diff
@@ -0,0 +1,91 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>Amy Cash: Blog</title>
+  <link rel="stylesheet" href="../stylesheets/normalize.css">
+  <link rel="stylesheet" href="../stylesheets/main.css">
+</head>
+
+
+<body>
+
+  <header>
+    <nav>
+      <div >
+        <ul class="left">
+          <li><a href="index.html" target="_blank">Home</a></li>
+          <li><a href="portfolio.html" target="_blank">Portfolio</a></li>
+          <li><a href="blog.html" target="_blank">Blog</a></li>
+          <li><a href="about.html" target="_blank">About</a></li>
+        </ul>
+      </div>
+
+      <div>
+        <ul  class="right">
+          <li><a href="https://twitter.com/cashmonger?lang=en"><img src=../images/contact-links/twitter.png /></a></li>
+          <li><a href="https://www.linkedin.com/in/amy-cash-0b5305149/"><img src=../images/contact-links/linkedin.png /></a></li>
+          <li><a href="https://www.linkedin.com/in/amy-cash-0b5305149/"><img src=../images/contact-links/email.png /></a></li>
+        </ul>
+      </div>
+
+    </nav>
+  </header>
+
+  <section class="blog-content">
```

Comment:
To maintain consistency with the About page we should surround this `section` in a `main` tag:
```html
<main>
  <section class="blog-content">
    ...
  </section>
</main>
```

Same goes for `index.html` and `portfolio.html`.